strategies/lwf/experiment.py

```diff
@@ -100,7 +100,7 @@ def test_pmnist(self, override_args=None):
         if args.check and target_acc > avg_stream_acc:
             self.assertAlmostEqual(target_acc, avg_stream_acc, delta=0.02)
 
-    def test_stinyimagenet(self, override_args=None):
+    def test_stinyimagenet(self, override_args=None, dataset_root=None):
```

Comment:
can you move the `dataset_root` argument inside the dictionary of arguments inside the function?